singer_sdk/tap_base.py

```diff
@@ -180,6 +182,28 @@ def capabilities(self) -> list[CapabilitiesEnum]:
             PluginCapabilities.FLATTENING,
         ]
 
+    @classmethod
+    def append_builtin_config(cls: type[PluginBase], config_jsonschema: dict) -> None:
+        """Appends built-in config to `config_jsonschema` if not already set.
+
+        To customize or disable this behavior, developers may either override this class
+        method or override the `capabilities` property to disabled any unwanted
+        built-in capabilities.
+
+        For all except very advanced use cases, we recommend leaving these
+        implementations "as-is", since this provides the most choice to users and is
+        the most "future proof" in terms of taking advantage of built-in capabilities
+        which may be added in the future.
+
+        Args:
+            config_jsonschema: [description]
+        """
+        super().append_builtin_config(config_jsonschema)
```

Comment:
The `mypy` complaint is on this line. Error is:

> singer_sdk/tap_base.py:201: error: Argument 2 for "super" not an instance of argument 1  [misc]

Probably fixed by using a more specific `super()` invocation.

Reply:
@aaronsteers Yeah, that was it 🙂

Reply:
Oh, I see! Thanks @edgarrmondragon for fixing that! 👍 🚀